<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Поле для ввода</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат операции</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерация ключей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generateAES">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерация ключа AES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_generateRSA">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Генерация ключевой пары</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранение шифра</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveToFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить  в файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AES_decrypt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AES (дешифрование)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AES_encrypt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>AES (шифрование)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RSA_decrypt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RSA (дешифрование)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RSA_encrypt">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RSA (шифрование)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>211</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Шифрование/дешифрование</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>241</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Окно для служебной информации</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_tech">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_toLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
